.gitignore

```diff
@@ -20,3 +20,8 @@ main
 dist/
 ddosify
 .vscode
+.idea/.gitignore
```

Comment:
Only `.idea/` should be enough, could you please change that?

Reply:
You are right, I've fixed it.